src/main/plugins/org.dita.pdf2/xsl/fo/links.xsl

```diff
@@ -407,9 +406,7 @@ See the accompanying license.txt file for applicable licenses.
           <fo:list-item xsl:use-attribute-sets="related-links.ul.li">
             <fo:list-item-label xsl:use-attribute-sets="related-links.ul.li__label">
               <fo:block xsl:use-attribute-sets="related-links.ul.li__label__content">
-                <fo:inline>
```

Comment:
In the end - there were convenient + appropriate group elements for almost everything. Syntax groups did not have group elements but it was easy / not disruptive to add them. 

The exceptions were `indexterm` and `imagemap`. For index term: with all the other processing that goes on, IDs are already added, so the `<fo:inline>` was useless, and I've removed it.

For image maps, I just moved the test into the `imagemap` template. Creating a new `fo:inline` would be wrong, and a new `fo:block` wrapper might mess up formatting for smaller images that appear inline.